tracing/spec/src/main/asciidoc/microprofile-telemetry-tracing-spec.asciidoc

```diff
@@ -60,13 +60,15 @@ https://opentelemetry.io[OpenTelemetry] is a set of APIs, SDKs, tooling and inte
 
 This specification defines the behaviors that allow MicroProfile applications to easily participate in an environment where distributed tracing is enabled via https://opentelemetry.io[OpenTelemetry] (a merger between https://opentracing.io[OpenTracing] and https://opencensus.io[OpenCensus]).
 
-This document and implementations *MUST* comply with the following OpenTelemetry {otel-spec-main-version} specifications:
+The OpenTelemetry specification describes the cross-language requirements and expectations for all OpenTelemetry implementations. This specification is based on the https://github.com/open-telemetry/opentelemetry-java/tree/v{otel-java-version}[Java implementation v{otel-java-version}] of OpenTelemetry. An implementation of this MicroProfile Telemetry may consume a later patch release of the Java implemenation as long as the required TCKs pass successfully.
 
-* https://github.com/open-telemetry/opentelemetry-specification/blob/v{otel-spec-version}/specification/overview.md[OpenTelemetry Overview]
-* https://github.com/open-telemetry/opentelemetry-specification/blob/v{otel-spec-version}/specification/trace/api.md[Tracing API]
-* https://github.com/open-telemetry/opentelemetry-specification/blob/v{otel-spec-version}/specification/baggage/api.md[Baggage API]
-* https://github.com/open-telemetry/opentelemetry-specification/tree/v{otel-spec-version}/specification/context[Context API]
-* https://github.com/open-telemetry/opentelemetry-specification/blob/v{otel-spec-version}/specification/resource/sdk.md[Resource SDK]
+Refer to the OpenTelemetry specification repo to understand some essential terms.
+
+* https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/overview.md[OpenTelemetry Overview]
+* https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md[Tracing API]
+* https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/baggage/api.md[Baggage API]
+* https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/context[Context API]
+* https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/sdk.md[Resource SDK]
```

Comment:
```suggestion
Refer to the OpenTelemetry specification repo to understand some essential terms.

* https://opentelemetry.io/docs/specs/otel/overview/[OpenTelemetry Overview]
* https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md[Tracing API]
* https://opentelemetry.io/docs/specs/otel/baggage/api/[Baggage API]
* https://opentelemetry.io/docs/specs/otel/context/[Context API]
* https://opentelemetry.io/docs/specs/otel/resource/sdk/[Resource SDK]
```

If we're linking to the latest version, we may as well link to the nicer version on the OTel website.